README.md

```diff
@@ -37,7 +37,7 @@ if let Ok(res) = res {
 let model = "llama2:latest".to_string();
 let prompt = "Why is the sky blue?".to_string();
 
-let mut stream = ollama.generate_stream(GenerationRequest::new(model, prompt)).await.unwrap();
+let mut stream = ollama.generate_stream(GenerationRequest::new(model, prompt,None)).await.unwrap();
```

Comment:
I think the format should be None by default without having to add it to the new method, like the other optional fields, as it meets a very particular need.